### PR TITLE
fix policy search

### PR DESCRIPTION
**Pull request by briskt**
### Fixed
- Fixed the policy search feature. It was not including policies with no users.

---

### Code Checklist
- [x] Documentation (README, goswagger, etc.)
- [x] Unit tests created or updated
- [x] Run `gofmt`
- [x] Run `make swaggerspec`

[CVR-777](https://itse.youtrack.cloud/issue/CVR-777)

### CODE REVIEW

File: application/models/policy.go [hunk 1]
```diff
@@ -388,13 +388,13 @@ func scopeSearchPolicies(searchText string) pop.ScopeFunc {
 	return func(q *pop.Query) *pop.Query {
 		return q.Where("policies.id IN ("+
```
Comment (jason-jackson) [inline application/models/policy.go:389]: Would this be better? (And does it still work?)
```suggestion
		return q.RawQuery("policies.id IN ("+
```

**Review by jason-jackson — APPROVED** [review]
Looks good. It would be nice if we could have used `q.LeftJoin()` in the scope function, but that didn't work (and I'm not certain it should work either given how pop works, meaning I think it's a feature, not a bug).

File: application/models/policy.go (hunk 1 above)
Comment (briskt) [reply application/models/policy.go:389]: No, it doesn't work. It appears to replace the entire query with only the text passed to RawQuery.

Comment (briskt) [reply application/models/policy.go:389]: The function doc for RawQuery:
```
// RawQuery will override the query building feature of Pop and will use
// whatever query you want to execute against the `Connection`. You can continue
// to use the `?` argument syntax.
```

**Pull request merged**